docs/porting-addons-to-v2.md

```diff
@@ -258,7 +258,13 @@ module.exports = addonV1Shim(__dirname);
       "version": 2
     }
     ```
-16. In the `addon` directory, run `yarn start` to start building the addon.
-17. In a separate shell, you should be able to go into the `test-app` directory and run `yarn start` or `yarn test` and see your tests passing.
 
+16. Now we are ready to build the addon. In the `addon` directory, run `yarn start`. This will compile your authored code into the `dist/` folder in v2 format.
```

Comment:
```suggestion
16. Now we are ready to build the addon. In the `addon` directory, run `npm start`. This will compile your authored code into the `dist/` folder in v2 format.
```


I see that this document previously mentions yarn.

We should probably remove that altogether, as yarn@v1 often has a rough time with correctly resolving dependencies :thinking:

Reply:
oh ok, there is a lil disclaimer here at the top, nevermind

> Our recommended way to manage these multiple packages is using a monorepo, via Yarn, NPM, or PNPM Workspaces. The example in this guide assumes a Yarn Workspaces monorepo.